cli/deployments/eirini.go

```diff
@@ -97,6 +78,7 @@ func (k Eirini) Delete(c *kubernetes.Cluster, ui *ui.UI) error {
 		}
 		if warning != "" {
 			ui.Exclamation().Msg(warning)
+			return nil
```

Comment:
Warnings usually do not exit the method, maybe a comment explaining this behavior might help

Reply:
Added a comment and exiting only for a specific warning.